lib/iris/fileformats/pp.py

```diff
@@ -606,6 +606,10 @@ def fill_value(self):
     def ndim(self):
         return len(self.shape)
 
+    @property
+    def meta(self):
```

Comment:
I suggest rename to "dask_meta" method, as for the NetCDFDataProxy one.

Reply:
Done in 00af67e